### ISSUE TITLE
[rush] unmet peer dependencies is failing the `rush update` link step

### ISSUE THREAD

**Issue by wbern** [microsoft/rushstack#1589]
## Please prefix the issue title with the project name i.e. [rush], [api-extractor] etc. ##

**Is this a feature or a bug?**

- [ ] Feature
- [X] Bug

**Please describe the actual behavior.**
I just ported a repo over to a rush repo and it is complaining about unmet peer dependencies. I don't have the "strictPeerDependencies" present in rush.json.

![image](https://user-images.githubusercontent.com/25477382/66852249-d6308d00-ef7c-11e9-8f20-e0e8bf0139e2.png)

**If the issue is a bug, how can we reproduce it? Please provide detailed steps and include a GitHub branch if applicable. Your issue will get resolved faster if you can make it easy to investigate.**

**What is the expected behavior?**
failed peer deps shouldn't fail the update/link step.

**If this is a bug, please provide the tool version, Node.js version, and OS.**

* **Tool:**
Rush 
* **Tool Version: **
rush 5.15.1 / pnpm 3.8.1
* **Node Version:**
10.15.3
  * **Is this a LTS version?**
Yes
  * **Have you tested on a LTS version?**
Yes 
* **OS:**
Windows 10

**Comment by iclanton** [comment]
Do you have a repro you can share?

**Comment by wbern** [comment]
@iclanton @rakeshpatnaik 
1. `git clone --single-branch --branch rush-issue-1589 https://github.com/wbern/rush-pnpm-issues.git`
2. Do `rush update`

**Comment by christiango** [comment]
Seeing the same issue inside of one of internal repos for Office when we tried to upgrade to the latest Rush. Setting 
```
  "pnpmOptions": {
    "strictPeerDependencies": false
  },
```
Did not seem to help.

**Comment by christiango** [comment]
Seems like a regression from https://github.com/microsoft/rushstack/pull/1526. We do have an unmet peer dependency, but 'ts from a dependency outside our control and we know it has no user bad (it's for isomorphic-ws's websocket polyfill for node and our code runs in the browser).

**Comment by D4N14L** [comment]
The change was merged. @christiango  @wbern could you confirm that this resolves your issue?

**Comment by iclanton** [comment]
This has been fixed in Rush 5.16.1.

**Comment by christiango** [comment]
Thanks! I was able to confirm that we can now install with the latest rush and pnpm
